lib/pom.xml

```diff
@@ -1254,7 +1254,7 @@ DO NOT MODIFY - GENERATED CODE
           <include>gems/timeout-0.4.3*/**/*</include>
           <include>gems/tsort-0.2.0*/**/*</include>
           <include>gems/un-0.3.0*/**/*</include>
-          <include>gems/uri-1.0.2*/**/*</include>
+          <include>gems/uri-1.0.3*/**/*</include>
```

Comment:
The header of pom.xml says "DO NOT MODIFY - GENERATED CODE". I think this file is not supposed to update.

Reply:
Thanks @kaisecheng , fixed